### DISCUSSION TITLE
Issue description in Markdown

### DISCUSSION THREAD

**Discussion by matthiasa4** [ankitpokhrel/jira-cli#617] [Ideas]
First of all - great tool that helps me stay sane and avoid the JIRA UI altogether.

I am embarking on a new project and I was thinking the following feature would be useful: bulk issue creation enabled by making the full issue description possible in a yaml file. This would include the things usually specified as flags in the `jira` command: `-tBug -s"Test" -yHigh -a test@test.com --template test.md`.

The idea would be to have a YAML-style issue description in the style of:
```
type: Bug
summary: Test
priority: High
assignee: test@test.com
template: /path/to/template.md
```

with the potential option of having the issue description inline in the YAML file as well. 

I didn't find this functionality yet, and it would be useful to either 1) script all the issues to create 2) even include a `jira create issue bulk` pointing to a folder with said YAML files.

Thoughts? :)

**Comment by codekiln** [comment]
Love this, just wanted to mention that I filed a discussion for the mirror of this, being able to render an existing JIRA issue as Markdown, in [Add `jira issue view PD-1234 --md` · ankitpokhrel/jira-cli · Discussion #823](https://github.com/ankitpokhrel/jira-cli/discussions/823)
